14.0.Dockerfile

```
@@ -0,0 +1,228 @@
+FROM python:3.6-slim-buster AS base
+
+EXPOSE 8069 8072
+
+ARG GEOIP_UPDATER_VERSION=4.3.0
+ARG WKHTMLTOPDF_VERSION=0.12.5
+ARG WKHTMLTOPDF_CHECKSUM='1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb'
+ENV DB_FILTER=.* \
+    DEPTH_DEFAULT=1 \
+    DEPTH_MERGE=100 \
+    EMAIL=https://hub.docker.com/r/tecnativa/odoo \
+    GEOIP_ACCOUNT_ID="" \
+    GEOIP_LICENSE_KEY="" \
+    GIT_AUTHOR_NAME=docker-odoo \
+    INITIAL_LANG="" \
+    LC_ALL=C.UTF-8 \
+    LIST_DB=false \
+    NODE_PATH=/usr/local/lib/node_modules:/usr/lib/node_modules \
+    OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
+    PATH="/home/odoo/.local/bin:$PATH" \
+    PIP_NO_CACHE_DIR=0 \
+    DEBUGPY_ARGS="--listen 0.0.0.0:5678 --wait-for-client" \
```

As discussed in https://github.com/Tecnativa/doodba/pull/324, and with the latest version there, we will still use port 6899
```suggestion
    DEBUGPY_ARGS="--listen 0.0.0.0:6899 --wait-for-client" \
```